docs/install_guides/installing-openvino-macos-header.md

```diff
@@ -13,9 +13,10 @@
    :hidden:
 
    From Archive <openvino_docs_install_guides_installing_openvino_from_archive_macos>
+   Using Homebrew <openvino_docs_install_guides_installing_openvino_brew>
    From PyPI <openvino_docs_install_guides_installing_openvino_pip>
    Using Conda Forge <openvino_docs_install_guides_installing_openvino_conda>
-   Using Homebrew <openvino_docs_install_guides_installing_openvino_brew>
+   Use VCPKG <openvino_docs_install_guides_installing_openvino_vcpkg>
```

Comment:
`vcpkg` is written with small letters in MSFT documentation. I suppose we should be aligned with the rest of community.